README.md

```diff
@@ -0,0 +1,50 @@
+Piinguin lambda relay
+=====================
```

Comment:
This is ReST heading, not Markdown, I'm even surprised this syntax works after they broke everything with CommonMark.